hw/ip/i2c/doc/_index.md

```diff
@@ -1,3 +1,225 @@
+
 ---
-title: "I2C Technical Specification" 
+title: "I2C HWIP Technical Specification"
 ---
+
+
+# Overview 
+
+This document specifies I2C hardware IP functionality.
+This module conforms to the [Comportable guideline for peripheral functionality.]({{< relref "doc/rm/comportability_specification/index.md" >}})
+See that document for integration overview within the broader top level system.
+
+
+## Features 
+
+- Two-pin clock-data parallel bidirectional external interface 
+- The initial revision only supports I2C as a Host ("I2C Master"<sup>1</sup>).
+- Support for standard (100 kbaud), fast (400 kbaud) and fast-plus (1 Mbaud) modes
+- Bandwidth up to 1 Mbaud
+- Support for all "Mandatory" features as specified for I2C Hosts (as listed in Table 2 of the [I2C specification](https://www.nxp.com/docs/en/user-guide/UM10204.pdf)): 
+    - Start Condition
+    - Stop Condition
+    - Acknowledge
+    - 7-bit device address
+- Support for the following optional capabilities:
+    - 10-bit device addressing
+    - Clock stretching (may be required by some devices)
+- *No support at this time* for I2C Target ("I2C Slave"<sup>1</sup>) functionality
+- *No support at this time* for any of the features related to multi-host control:
+    - No support at this time for any of the features related to multi-host control:  
```

Comment:
seems like this line is just a duplicate of the above?

Reply:
Oh, good catch.   Thanks.